### ISSUE TITLE
Can we please see a Fedora RPM be created? 

### ISSUE THREAD

**Issue by SpaceRaccoonFluff** [elesiuta/picosnitch#22]
I'm just a Linux noob in desperate search to permanently replace the god awful creation that is Windows and a bandwidth monitor is the final piece of critical software that's missing. Glasswire on Windows is so simple and beautiful at monitoring per app bandwidth and your Picosnitch is the closet thing I've come across for Linux after much too many hours searching. Portmaster is a great new per app firewall but it's not designed to be for bandwidth monitoring and not something they've said they want to support either. Being able to use Picosnitch to see whats hogging up too much bandwidth and then using Portmaster to block that specific thing with ease is a very beautiful enticing combo that I would love to have on Linux. 

Unfortunately though there's no easy way to download Picosnitch on Fedora. This is really really sad as Picosnitch is perfect for newer users like myself looking for a simple and elegant alternative to Glasswire at a time when many more newer Linux users are going to be giving Fedora a try instead of Ubuntu as one of their first distro's. https://www.youtube.com/watch?v=D9h_0dnSGWk and https://www.makeuseof.com/reasons-fedora-is-new-ubuntu/. 

As a linux noob just wanting to not be in the dark on what my new operating system with new apps are up to, I've had to sadly and frustratingly give on installing a bandwidth monitor until I can find a simpler way to install one. Having tried the PyPi method, I created quite a mess in the command line and unsure on how to exactly uninstall either. It would be tremulously helpful for all the new Linux users on Fedora if instead a RPM installation option was available as well. Much thanks in advance if you could spare the time to create one.

**Comment by elesiuta** [comment]
I would like to have a package and repository for Fedora, unfortunately I don't have time to look into it myself. If there's anyone who knows how or is willing to figure it out, I'd happily review it and add it here.

It looks like I can use https://copr.fedorainfracloud.org/ to host it pretty easily as long as someone can provide me with an RPM SPEC file and the GitHub actions. I'm also willing to use https://build.opensuse.org/ or https://packagecloud.io/l/yum-repository if that's easier.

Basically I want the packaging to be completely automated and transparent, i.e. you're not packaging it on your own machine and uploading it. For example, the deb for the ppa is packaged entirely on canonical's servers, and the python package and tar for arch are done with GitHub actions and hashed to reduce the number of parties you need to trust and make it easier to verify the package matches the source.

**Comment by elesiuta** [comment]
There is now a fedora package you can install from [copr](https://copr.fedorainfracloud.org/coprs/elesiuta/picosnitch/) !
